packages/access-api/src/config.js

```diff
@@ -35,6 +35,12 @@ export function loadConfig(env) {
     }
   }
 
+  if (typeof env.ACCESS_API_R2 !== 'object') {
+    throw new TypeError(
+      `expected env.ACCESS_API_R2 to be an R2Bucket object, but got ${typeof env.ACCESS_API_R2}`
```

Comment:
it shouldn't happen under normal circumstances, i.e. there is no wrangler magic to recommend. the `wranger.toml` will set this up in a good way for known envs. This would catch it if you somehow opted out of that calling this function.